### ISSUE TITLE
"keybase chat send" returns error

### ISSUE THREAD

**Issue by hefaistion** [keybase/client#5635]
Another problem w/ chat here. I've been having issues using chat with a specific user while being able to use it with other users just fine.

What I get after 
`keybase chat send user "message"`
is 
`Using private conversation me,user.`
`▶ ERROR no valid message in the conversation`

log ID is: 7c9e825539c632c7393a3d1c

Thought it might interest you. Thx & keep up the good work.

**Comment by maxtaco** [comment]
cc: @mmaxim

**Comment by mmaxim** [comment]
It looks like the person you were talking to reset their account at some point, which is the source of the problem (that flow for us has been tricky to get right). I'll keep looking and see why it is failing there. Sorry!

**Comment by hefaistion** [comment]
Yes, the other user reset their account. Its not bothering me, but I thought you might wanna know about that kind of bug.
Thanks for the reply!
